include/aspect/simulator.h

```diff
@@ -1043,7 +1062,7 @@ namespace aspect
        */
       void
       compute_material_model_input_values (const LinearAlgebra::BlockVector                            &input_solution,
-                                           const FEValues<dim,dim>                                     &input_finite_element_values,
+                                           const FEValuesBase<dim,dim>                                     &input_finite_element_values,
```

Comment:
I think this change will be merged into the developer version with #723 soon anyway, will be removed during the rebase before merging.